sql/planbuilder/show.go

```diff
@@ -615,6 +615,8 @@ func (b *Builder) buildAsOfExpr(inScope *scope, time ast.Expr) sql.Expression {
 			err := sql.ErrInvalidAsOfExpression.New(v)
 			b.handleErr(err)
 		}
+	case *ast.Subquery:
```

Comment:
is there anyway we could easily extend this a bit? should the default be an error? would we need a whitelist of acceptable exprs? longer blacklist of non-acceptable exprs?